---
source: crates/ruff_linter/src/rules/flake8_bugbear/mod.rs
---
B029 Using `except ():` with an empty tuple does not catch anything; add exceptions to handle
  --> B029.py:8:1
   |
 6 |   try:
 7 |       pass
 8 | / except ():
 9 | |     pass
   | |________^
10 |
11 |   try:
   |

B029 Using `except ():` with an empty tuple does not catch anything; add exceptions to handle
  --> B029.py:13:1
   |
11 |   try:
12 |       pass
13 | / except () as e:
14 | |     pass
   | |________^
15 |
16 |   try:
   |

B029 Using `except* ():` with an empty tuple does not catch anything; add exceptions to handle
  --> B029.py:18:1
   |
16 |   try:
17 |       pass
18 | / except* ():
19 | |     pass
   | |________^
20 |
21 |   try:
   |

B029 Using `except* ():` with an empty tuple does not catch anything; add exceptions to handle
  --> B029.py:23:1
   |
21 |   try:
22 |       pass
23 | / except* () as e:
24 | |     pass
   | |________^
   |
